### DISCUSSION TITLE
Does renovate supports Gradle handled npm dependency updates?

### DISCUSSION THREAD

**Discussion by warnyul** [renovatebot/renovate#18078] [Request Help]
When I define npm dependencies in a `build.gradle` file, when I am using Kotlin Multiplatform projects. Can renovate recognize the npm dependencies, and check for new versions?

Sample code in the link:
https://kotlinlang.org/docs/using-packages-from-npm.html

**Comment by rarkins** [comment]
No, I don't think we support this yet. It is documented to only extract Maven dependencies so far: https://github.com/renovatebot/renovate/blob/1e9dc34c545ff5592799bfc27f8ed2115d29855b/lib/modules/manager/gradle/index.ts#L21

Please create a reproduction repo and then raise a feature request issue
